torch/csrc/jit/fuser/cpu/fused_kernel.cpp

```diff
@@ -16,9 +20,31 @@ namespace jit {
 namespace fuser {
 namespace cpu {
 
+#ifdef _MSC_VER
+static const std::string getTempPath() {
+  char lpTempPathBuffer[MAX_PATH];
```

Comment:
Technically, shouldn't this be `MAX_PATH+1`, because `MAX_PATH` doesn't include the trailing null terminator? (I'm a Windows API noob so please let mek now if I'm wrong.)

Reply:
MAX_PATH = 256, the max length for PATH is 255, so it is no need to use `MAX_PATH+1`.